histoDB/histodb-dataserver/pom.xml

```diff
@@ -137,6 +137,7 @@
         <dependency>
             <groupId>eu.itesla_project</groupId>
             <artifactId>cim1-import</artifactId>
+            <version>0.1-SNAPSHOT</version>
```

Comment:
Anyway, in the latest commit cim-import  and iidm-xxxx version infos are now declared in histodb's top pom.xml